src/App.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import { TextField } from '@material-ui/core';
+
+import './App.css'
+
+
+
+class App extends React.Component {
+    constructor(props) {
+      super(props);
+      this.state = {value: ''};
+  
+      this.handleChange = this.handleChange.bind(this);
+      this.handleSubmit = this.handleSubmit.bind(this);
+    }
+  
+    handleChange(event) {
+      this.setState({value: event.target.value});
+    }
+  
+    handleSubmit(event) {
+      console.log(this.state.value);
+      event.preventDefault();
+    }
+  
+    render() {
+      return (
+        <form onSubmit={this.handleSubmit}>
+          <label>
```

Comment:
"label" tag here is redundant - it's included in TextField MaterialUI component.
See the TextField prop called "label".

Reply:
done